src/model/Model.lua

```diff
@@ -168,7 +168,32 @@ end
      * @return the model text.
 ]]
 function Model:saveModelToText()
-
+	local tokenPatterns={}
```

Comment:
If possible please try to follow the current 4-space indentation so that the style remains consistent.